miqa/core/rest/scan.py

```diff
@@ -5,12 +5,15 @@
 
 from miqa.core.models import Scan
 
+from .scan_note import ScanNoteSerializer
```

Comment:
This could lead to circular import issues, but we can cross that bridge when we come to it.